packages/carbon-react/src/components/TextInput/TextInput.stories.js

```diff
@@ -0,0 +1,221 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { useState } from 'react';
+import { action } from '@storybook/addon-actions';
+import { withKnobs, boolean, select, text } from '@storybook/addon-knobs';
```

Comment:
Do you get errors in the console from this since `addon-knobs` is deprecated/removed in v6? I was unable to get DatePicker stories working with the knobs functions in the props object/functions. I ended up stripping them all down to the default values from the knob params, like this:

```diff
-defaultValue: text(
-      'Default value (defaultValue)',
-      'This is not a default value'
-    ),
+ defaultValue: 'This is not a default value',
```

Reply:
Nope. No console errors.